saleor/settings.py

```diff
@@ -83,9 +83,9 @@ def get_bool_from_env(name, default_value):
     ('zh-hans', _('Chinese')),
     ('zh-tw', _('Chinese (Taiwan)'))]
 LOCALE_PATHS = [os.path.join(PROJECT_ROOT, 'locale')]
-USE_I18N = True
-USE_L10N = True
-USE_TZ = True
```

Comment:
Should this not be `False` by default. See docs here. https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-USE_TZ